flink-table/flink-table-common/src/test/java/org/apache/flink/table/types/LogicalTypesTest.java

```diff
@@ -797,42 +831,46 @@ private StructuredType createHumanType(boolean useDifferentImplementation) {
                         useDifferentImplementation ? SpecialHuman.class : Human.class)
                 .attributes(
                         Collections.singletonList(
-                                new StructuredType.StructuredAttribute(
-                                        "name", UDT_NAME_TYPE, "Description.")))
+                                new StructuredAttribute("name", UDT_NAME_TYPE, "Description.")))
                 .description("Human type desc.")
                 .setFinal(false)
                 .setInstantiable(false)
                 .build();
     }
 
-    private StructuredType createUserType(boolean isRegistered, boolean isFinal) {
+    private StructuredType createUserType(
```

Comment:
should it be a builder with more selfexplainable methods?
Asking since now `createUserType(false, false, false)` without looking into implementation it is hard to what all those `false` mean